cleanlab/datalab/internal/issue_finder.py

```diff
@@ -343,6 +355,17 @@ def get_available_issue_types(self, **kwargs):
             warnings.warn("No labels were provided. " "The 'label' issue type will not be run.")
             issue_types_copy.pop("label")
 
+        label_use_trained_statistics = (
+            "label" in issue_types_copy
+            and self.datalab.has_trained_statistics
+            and not drop_label_check
+        )
+        if label_use_trained_statistics:
+            warnings.warn("Using precomputed statistics from a previously trained datalab. ")
+            issue_types_copy = self._resolve_trained_statistics_args(
+                issue_types_copy, kwargs=kwargs
+            )
```

Comment:
I may need to test the different scenarios, but this is the first time we're adding a helper function that works with the `**kwargs`.

Maybe document more closely in the function what keyword arguments we want to use here.